dracut/modules.d/90kiwi-dump/kiwi-dump-image.sh

```diff
@@ -34,6 +34,22 @@ function scan_multipath_devices {
     systemctl start multipathd
 }
 
+function binsize_to_bytesize {
```

Comment:
This is great but imho should be a method of the lib. Would you consider moving it to ```99kiwi-lib/kiwi-lib.sh```